config/core/rootfs-configs.yaml

```diff
@@ -13,6 +13,87 @@ rootfs_configs:
     frags:
       - baseline
 
+  bullseye:
+    rootfs_type: debos
+    debian_release: bullseye
+    arch_list:
+      - amd64
+      - arm64
+      - armel
+      - armhf
+      - i386
+      - mips
+      - mips64el
+      - mipsel
+    extra_packages:
+      - isc-dhcp-client
+    extra_packages_remove: &extra_packages_remove_bullseye
+      - bash
+      - e2fslibs
+      - e2fsprogs
+      - klibc-utils
+      - libext2fs2
+      - libgnutls30
+      - libklibc
+      - libncursesw6
+      - libp11-kit0
+      - libunistring2
+      - sensible-utils
+    extra_files_remove: &extra_files_remove_bullseye
+      - '*networkd*'
+      - '*resolved*'
+      - nosuspend.conf
+      - tar
+      - patch
+      - dir
+      - partx
+      - find
+    script: "scripts/install-bootrr.sh"
+    test_overlay: "overlays/baseline"
+
+  bullseye-igt:
```

Comment:
Please split this to a separate commit and make another PR on top of a base one with only the plain `bullseye`.  Then we'll be able to first test just the bullseye update before starting to enable test suites on top of it.

Reply:
Okay removed igt part from this PR. Now it has only `plain bullseye`  rootfs configuration.